understanding/21/reflow.html

```diff
@@ -12,68 +12,245 @@ <h1>Understanding Reflow</h1>
+          <figcaption>
+            <p>A website provides a “carousel” or “swim-lane” of different panels of teaser (a.k.a., “lead-in”) content.This content container allows for horizontal scrolling, independent from the vertical scrolling of the primary web page, and the vertical direction of reading of the individual sections of content within this horizontally scrolling container.</p>
+          </figcaption>
+        </figure>
+
+        <figure>
+          <img src="img/single-carousel-panel.png" alt="one of the carousel panels zoomed in 400%. The text vertically flowing text fits within 320px width container." width="550">
+          <figcaption>
+            <p>The content of each carousel "panel" (image, text content and call to action link visually represented as a right-pointing arrow icon) fit within a width of 320 CSS pixels. This helps ensure a user need only scroll vertically to read the text of each section of teaser content. The user can horizontally scroll the wrapping container element to bring each width-conforming panel of teaser content into view.  A user can scroll vertically to view the image or read the text of each section of content.</p>
+          </figcaption>
+        </figure>
       </section>
+  
       <section>
-      <h3>Responsive web design and other ways to meet this Success Criterion</h3>
-      <p>Using the responsive web design approach is the most effective method of achieving the goal of allowing people to zoom in to 400%. Each variation (CSS break point) of the page at the same URL should conform (compare <a href="https://www.w3.org/TR/WCAG21/#h-note-27">Conformance for WCAG 2.1</a>).</p>
-      <p>For organisations which are using legacy systems or are not able to update their layout methods for some reason, an alternative conforming version could be a mobile site which has a fixed 320px wide layout. The user should be able to find that version from the default website.</p>
+        <h3>Exceptions and content considerations for Reflow</h3>
+        <p>For many 'traditional' web pages where the page's primary purpose is for a user to read distinct <a href="https://www.w3.org/TR/WCAG/#dfn-section">sections of content</a> containing multi-line paragraphs of text, the web page is generally expected to scroll vertically (when written in languages such as English). For users with low vision who might need to zoom the web page to read, it is generally expected that as the viewport size decreases due to the increase in zoom level, that the content of the page would reflow to fit within the confines of the viewport.</p>
+        
+        <p>However, not all content can fully reflow without degradation of the information or functionality the content represents - <a href="#note-2dlayout">as noted in the Reflow success criterion</a>. Examples are tabular data, graphics, maps, presentations, or interfaces that necessiate persistent toolbars.</p>
```

Comment:
I'm not sure this is necessary, as we already understand the Reflow success criterion. Is the anchor really needed? When I first read this without interacting, I felt like I was in the wrong place. Additionally, I’m unclear on where this link is pointing (the anchor doesn't work). If we're referring to the normative exception, it might be helpful to include phrasing like "as noted in the normative part of the Reflow success criterion" to clarify the anchor's purpose.

Reply:
I'm not opposed to the anchor; I’m just sharing my initial reaction to reading this text. Typically, when we use anchors to refer to SCs within a specific understanding, they point to other SCs. In this case, my first instinct was that I might be in the wrong section. That's why I suggested adding the word "normative" (e.g., "as noted in the normative part of the Reflow success criterion"), assuming it's referring to the normative portion at the top. I'm not entirely sure where this anchor is supposed to point.

Reply:
ok, thanks for re-explaining.  i get what you're saying now.  

It sure doesn't help that the fragment i tried to add isn't resolving in the preview.  unfortunately i think i shoudl remove the anchor.  Thanks for pointing this out!